scss/detail.scss

```diff
@@ -0,0 +1,33 @@
+.book {
+	color: $black;
+
+	&__title {
+		margin: 0 0 52px 0;
+		h1 {
```

Comment:
Avoid to use element name selector in your CSS, we try using classes. Check our Style Guide here https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md